controllers/defaults.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"os"
+)
+
+var (
+	DefaultValMap = map[string]string{
```

Comment:
```suggestion
	defaultValMap = map[string]string{
```
Looks like it won't be used outside this package.

Reply:
Done